projects/10up-theme/assets/css/frontend/testing.scss

```diff
@@ -1,5 +1,5 @@
 // Test comment
-@import "./components/sass-component";
+@import url("./components/sass-component.scss");
```

Comment:
@nicholasio thoughts here? I think being consistent between `scss` and no `scss` is the way to go. Doesn't seem possible to have one linting rule _just_ for `scss` rules so it's one or the other.

Reply:
fine by me!